requirements.txt

```
@@ -2,4 +2,4 @@ PyYAML~=5.4
 python-dateutil~=2.8
 jsonpointer~=2.0
 json-ref-dict>=0.6.2,<0.8.0
-typing_extensions~=3.7
+typing_extensions~=4.2.0
```

I think we can keep this very loose, so it still works with packages which depend on `typing_extensions<4.0.0`:

```suggestion
typing_extensions>=3.7.0,<5.0.0
```

---

This makes sense, thanks!